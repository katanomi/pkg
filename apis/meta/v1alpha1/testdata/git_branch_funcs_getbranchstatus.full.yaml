kind: GitBranch
apiVersion: v1alpha1
metadata:
  name: release-1.7
  creationTimestamp: null
spec:
  project: kubernetes
  repository: kubernetes
  name: release-1.7
  protected: true
  # not returning to have a defaulting logic
  # default: false
  commit:
    sha: 347f652944cc2348d59b1c815007e21e062fe597
    createdAt: null
  webURL: https://github.com/kubernetes/kubernetes/tree/release-1.7
  downloadURL:
    zip: https://github.com/kubernetes/kubernetes/archive/refs/heads/release-1.7.zip
    tar.gz: https://github.com/kubernetes/kubernetes/archive/refs/heads/release-1.7.tar.gz
    tar.ba2: null
    tar: null
  properties:
    webURL: https://github.com/kubernetes/kubernetes/tree/release-1.7
