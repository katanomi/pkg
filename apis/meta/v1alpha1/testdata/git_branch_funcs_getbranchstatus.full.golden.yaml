name: release-1.7
protected: true
default: false
webURL: https://github.com/kubernetes/kubernetes/tree/release-1.7

